packages/ckeditor5-watchdog/docs/features/watchdog.md

```diff
@@ -41,11 +41,11 @@ Then, change your `ClassicEditor.create()` call to `watchdog.create()` as follow
 
 ```js
 import { ClassicEditor } from '@ckeditor/ckeditor5-editor-classic';
-import { EditorWatchdog } from '@ckeditor/ckeditor5-watchdog';
+import { ClassicEditor, EditorWatchdog } from '@ckeditor/ckeditor5-watchdog';
 
-import { Bold, Italic } from '@ckeditor/ckeditor5-basic-styles';
-import { Essentials } from '@ckeditor/ckeditor5-essentials';
-import { Paragraph } from '@ckeditor/ckeditor5-paragraph';
+import { ClassicEditor, Bold, Italic } from '@ckeditor/ckeditor5-basic-styles';
+import { ClassicEditor, Essentials } from '@ckeditor/ckeditor5-essentials';
+import { ClassicEditor, Paragraph } from '@ckeditor/ckeditor5-paragraph';
```

Comment:
```suggestion
import { ClassicEditor, Bold, EditorWatchdog, Essentials, Italic, Paragraph } from 'ckeditor5';
```

I guess something went wrong here after some mechanical changes. I also guess that EditorWatchdog will be available in the open-source package.